### PR TITLE
 Pay for a tax, duty, levy or grant

### PR DESCRIPTION
**Pull request by Jon-Rowe-HMRC**

### CODE REVIEW

File: src/examples/pay-for-a-tax/bank-transfer/index.njk [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+---
+
+<h2 class="govuk-heading-l">Pay by bank transfer</h2>
+
+<p class="govuk-body">
+If you pay by CHAPS (Clearing House Automated Payment System) or Faster Payments, you can submit your payment on the same or next day.
+</p>
+
+<p class="govuk-body">
+If you pay by Bacs (Bankers Automated Clearing System), allow 3 working days for the payment to reach HMRC.
```
Comment (timsb) [inline src/examples/pay-for-a-tax/bank-transfer/index.njk:12]: Should `Bacs` be uppercase?

**Comment by Jon-Rowe-HMRC** [comment]
I thought that too, ‘Bacs’ seems to be the accepted style thoughOn Mar 30, 2023, at 3:56 PM, Timothy Bryan ***@***.***> wrote:﻿
@timsb commented on this pull request.



In src/examples/pay-for-a-tax/bank-transfer/index.njk:
> @@ -0,0 +1,61 @@
+---
+layout: layout-example.njk
+---
+
+<h2 class="govuk-heading-l">Pay by bank transfer</h2>
+
+<p class="govuk-body">
+If you pay by CHAPS (Clearing House Automated Payment System) or Faster Payments, you can submit your payment on the same or next day.
+</p>
+
+<p class="govuk-body">
+If you pay by Bacs (Bankers Automated Clearing System), allow 3 working days for the payment to reach HMRC.

Should Bacs be uppercase?

File: src/examples/pay-for-a-tax/bank-transfer/index.njk [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+<p class="govuk-body">
+You must make all payments in sterling (GBP).
+</p>
+
+<p class="govuk-body">
+If needed, you can provide your bank with the following HMRC banking address:
+</p>
+
+<p class="govuk-body" style="border-left: 1px solid #b1b4b6; padding-left: 15px; margin-bottom: 30px; margin-top: 30px;">
```
Comment (timsb) [inline src/examples/pay-for-a-tax/bank-transfer/index.njk:55]: Are the GOV.UK team able to produce this without adding in the CSS?

**Review by timsb — APPROVED** [review]

**Comment by Jon-Rowe-HMRC** [comment]
@timsb Have merged the development file at the same time as asking for your review as ideally need GOV.UK to preview this before publishing later today

**Pull request merged**